### DISCUSSION TITLE
Deploy custom client

### DISCUSSION THREAD

**Discussion by k2000vi** [rustdesk/rustdesk-server-pro#345] [Q&A]
When we use the custom clients build we cant specify the «  relay server » so the client don’t work.
is it possible to specify into th msi ?

![image](https://github.com/user-attachments/assets/2053f4a3-dc69-460c-bd9b-a28cb414e91b)

**Comment by rustdesk** [comment]
We do not recommend you set it on client side. https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/relay/

If you really want to do this, you can set `relay-server=example.com:21117` in advanced options of custom client.

**Reply by k2000vi** [reply]
Thanks :)
